recipes/sos-rmarkdown/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sos-rmarkdown" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d67523c57e41b522bef91078565b616bfceeec66b7d250acd965e73d479f743b
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
Please add `python >2.7` for both `host` and `run` requirements sections.

Reply:
Ohmm, you meant `noarch`, but with `python > 2.7` in `host` and `run`? I think `noarch` does not accept any selector.

Reply:
Yes, that is what I meant. And you can add selectors with `noarch` in the `requirements` section.

Reply:
Done.